currencies/rate_converter_test.go

```diff
@@ -0,0 +1,477 @@
+package currencies_test
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/prebid/prebid-server/currencies"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFetch_Success(t *testing.T) {
```

Comment:
I've seen a few race conditions already... but they're notoriously hard to pick out by eye. I trust good automated tests much more than myself.

Please add a test which exercises the concurrency. Name it something like `TestRace*()`, and spin up some goroutines which do a bunch of reads & updates in parallel. The CI runs tests which start with `TestRace` under the `-race` flag, so this would really help prevent prod bugs. Race conditions crash the entire process, so... it's extremely important that we catch these.

Reply:
Yes !

Reply:
Yep, good point

Reply:
Done, 50 workers running `Updates` and `GetRate`:)